.travis.yml

```diff
@@ -1,25 +1,9 @@
 sudo: false
 language: go
 go:
-  - 1.5.4
-  - 1.6.2
-  - tip
+  - 1.9
+  - "1.10"
```

Comment:
I'd suggest going with `"1.9.x"` and `"1.10.x". No reason not to use latest patch version of each.

Reply:
Yup, I didn't realize `1.9` wouldn't be doing that, I'll fix it.